.github/workflows/continuous-integration.yml

```diff
@@ -0,0 +1,56 @@
+name: Continuous Integration Workflow
```

Comment:
Have added CI here, will need to combine this with the one in test-API to avoid conflicts.